apps/opik-backend/src/main/java/com/comet/opik/api/Prompt.java

```diff
@@ -30,6 +31,8 @@ public record Prompt(
                 Prompt.View.Updatable.class}) @Pattern(regexp = NULL_OR_NOT_BLANK, message = "must not be blank") String description,
         @JsonView({
                 Prompt.View.Write.class}) @Pattern(regexp = NULL_OR_NOT_BLANK, message = "must not be blank") @Nullable String template,
+        @JsonView({Prompt.View.Write.class}) @Nullable JsonNode metadata,
```

Comment:
will update in a follow up PR to unblock FE sooner

Reply:
I checked and it turns out that we do need these annotations for `Prompt` object. I will remove for `PromptVersion`.
In case of `Prompt` removing them leads to following error:
```
parameter '[template, metadata, changeDescription]' has no matching columns in the result set. Verify that the Java compiler is configured to emit parameter names, that your result set has the columns expected, annotate the parameter names explicitly with @ColumnName, or annotate nullable parameters as @Nullable
! at org.jdbi.v3.core.mapper.reflect.ConstructorMapper.createSpecializedRowMapper(ConstructorMapper.java:268)
```